### PR TITLE
fix:  added function when one faq closes the other opens

### LINKED ISSUE #524: When another item is clicked on the FAQ previous opened item won't close.
**Summary:**

When another item is clicked on the FAQ previous opened item won't close. We need only one (or zero) expanded group at a time.

**Device and Android version:**

Device Real Me 2
Android Version 8.1.0

**Screenshots:**

Screenshot_2019-03-27-14-59-26-39

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #524 
Added function when one faq opens the other closes

- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please review

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java [hunk 1]
```diff
@@ -84,6 +87,20 @@ public void initObjects() {
 
     }
 
+    public void initExpand() {
+        expandableListView.setOnGroupExpandListener(
+                new ExpandableListView.OnGroupExpandListener() {
+                    int prevExpandPos = -1;
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]:
```suggestion
                    int previousExpandPosition = -1;
```
Keep variable names as descriptive as you can yet short.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java [hunk 2]
```diff
@@ -84,6 +87,20 @@ public void initObjects() {
 
     }
 
+    public void initExpand() {
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: Give a better name to this function.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java (hunk 1 above)
Comment (Bhavnaharitsa) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: Okay!

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Done sir. Please check now

**Comment by luckyman20** [comment]
Please squash the commits and fix the travis build.

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Can I keep the variable as prevExpandPosition  cause the Travis fails saying the line is longer than 100 characters

**Comment by luckyman20** [comment]
Okay. That works.

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Done sir

**Comment by Bhavnaharitsa** [comment]
@shiv07tiwari You mean when more FAQs will be added, the user should be able to scroll down the FAQs that he/she wants to see.

**Comment by luckyman20** [comment]
@Bhavnaharitsa Some answers can have long answers too. I am closing this PR. Create a new issue and PR.

**Comment by Bhavnaharitsa** [comment]
@luckyman20 You mean I will have to implement view scroll inside each answer?
I will open a new issue and pr but do you want any other changes than this?

**Comment by luckyman20** [comment]
@Bhavnaharitsa No, we do not want any other changes than this. You don't have to implement scroll bar inside answers but to whole FAQ view.

**Pull request closed without merging**